packages/interop-tests/src/__test__/interop.test.ts

```diff
@@ -186,14 +171,6 @@ it('server wallet creates channel + cooperates with browser wallet to fund chann
     },
     'dummyDomain'
   );
-
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const postFundA = await fromEvent<SingleChannelOutput>(serverWallet as any, 'channelUpdated')
-    .pipe(first(containsPostfundState))
-    .toPromise();
-
-  serverWallet.on('channelUpdated', e => console.log(JSON.stringify(e)));
-  await browserWallet.pushMessage(serverMessageToBrowserMessage(await postFundA), 'dummyDomain');
```

Comment:
Re. 4416ef139705671252c01782d15732dd6aac579b

This commit seems like it's changing the interop test to _no longer check that the direct funding protocol works_. Is that correct?

That definitely makes me feel uneasy!

Reply:
This PR does break direct funding by removing the `ChannelUpdated` event, which should get restored in #3549.

I'll create an issue for this test as well.

Reply:
> I feel pretty wary of intentionally breaking the _most important feature_ of a wallet in the `master` branch.
> 
> Does it make sense to _not_ merge this branch `ag/remove-all-events`, but instead leave it as the "development" branch for your pitch until directly funded channels are supported in that branch?

I agree that a feature branch would make sense here since I am breaking functionality here.

I'm going to create a branch `eventless-engine` that will serve as my feature branch. I'll still raise PRs for review into it but we can also do a review of all the changes at the end as well.